### DISCUSSION TITLE
Duration >24h shown as mod 24h

### DISCUSSION THREAD

**Discussion by felixguendling** [valhalla/valhalla#3474] [Q&A] [answered]
I'm trying the demo instance on [https://valhalla.openstreetmap.de/](https://valhalla.openstreetmap.de/). It seems like durations >24h are shown as modulo 24h. The JSON contains the correct time.
![Screenshot from 2021-12-25 21-28-55](https://user-images.githubusercontent.com/1915889/147393149-2c146448-8614-4de8-b1b9-3ad56e3ab7f5.png)

**Comment by nilsnolde** [comment, marked as answer]
Urgh, thanks! We’ll fix that. Or you can PR if you want. 

It’s not yet very visible (I just realized;)), but the frontend lives here: https://github.com/gis-ops/valhalla-app

**Reply by felixguendling** [reply]
I opened a PR here:
https://github.com/gis-ops/valhalla-app/pull/4

**Reply by TimMcCauley** [reply]
@felixguendling this has been fixed and deployed - thanks for highlighting again
<img width="1429" alt="Screenshot 2021-12-28 at 20 43 27" src="https://user-images.githubusercontent.com/10322094/147601574-e59ddbe5-8981-4713-87be-88e66b6df97b.png">
